_posts/2018-03-23-crossword-data-format.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: post
+title:  "Yet Another Crossword Data Format"
+date:   2018-03-23 10:30:30 +0100
+categories: Experiment
+teaser: Experiment
+excerpt: >
+  Defining a new data format for simple crosswords
+---
+When starting our interactive crossword project (link), we looked for an existing crossword data format, but not many were documented and widely used. Not wishing to create [Yet Another Standard](https://xkcd.com/927/), we went with [https://www.xwordinfo.com/JSON/](https://www.xwordinfo.com/JSON/) but it proved too error-prone, and forced a clumsy parse/build structure on the code. After several acrimonious intra-team discussions, we looked for ‘a better way’, and a new format was born.
```

Comment:
Add `"_blank"` targets to all external links

Reply:
done